frontend/src/metabase/styled-components/theme/css-variables.ts

```diff
@@ -81,6 +81,7 @@ function getDesignSystemCssVariables(theme: MantineTheme) {
     --mb-color-success: ${theme.fn.themeColor("success")};
     --mb-color-summarize: ${theme.fn.themeColor("summarize")};
     --mb-color-warning: ${theme.fn.themeColor("warning")};
+    --mb-color-background: ${theme.fn.themeColor("bg-white")};
```

Comment:
FYI, this would be fixed by https://github.com/metabase/metabase/pull/46650 if it's merged.

The CSS custom properties would be dynamically defined here
https://github.com/metabase/metabase/blob/f079e23a77acbf77276a9517fa2652c54053a86c/frontend/src/metabase/styled-components/theme/css-variables.ts#L62-L81